wachiranduati/Questioner-v2
# #163342434  setup travisci codeclimate coveralls

**What does this PR do?**
This will create the travis.yml and the coverall.yml files to integrate Questioner-v2 into the cli. 

**Description of Task to be completed?**
- Create the travis.yml file and Integrate the repository into travis
- Create the coveralls.yml  file and integrate the repository into coveralls

**How should this be manually tested?**
- git clone this repo 
> $ git clone https://github.com/wachiranduati/Questioner.git
- cd into it.
> $ cd Questioner
- switch into the develop branch
> $ git checkout develop
- Navigate into the root folder and open the README file in your preferred editor.
- Cli badges should be visible then.
- By visiting this [repo's](https://github.com/wachiranduati/Questioner-v2) develop branch page. You should be able to view the cli badges.

**What are the relevant pivotal tracker stories?**
#163342434 [This Story on PT](https://www.pivotaltracker.com/story/show/163342434)
